docs/use-case2.rst

```diff
@@ -11,63 +11,80 @@ component quite unhandy.
 
 .. image:: LARGER_SYSTEM_BAD.png
 
-:code:`Integration-service` eases this process, allowing to integrate any :code:`DDS` system into an already
+*eProsima Integration-Service* eases this process, allowing to integrate any *DDS* system into an already
 existing system or viceversa, by providing an out-of-the-box bridge that straightforwardly allows to communicate the 
-:code:`DDS` and the non-:code:`DDS` protocols.
+*DDS* and the non-*DDS* protocols.
 
-Also, the core of :code:`integration-service` allows to centralise all the possible bridges among
+Also, the core of *eProsima Integration-Service* allows to centralise all the possible bridges among
 the subsystems by means of system-specific plugins, or **System-Handles**, that speak the same language as the core.
 
-Once all the protocols of interest are communicated with :code:`integration-service`,
+Once all the protocols of interest are communicated with *eProsima Integration-Service*,
 each via a dedicated **System-Handle**, the inter-components communication
 can be easily implemented by means of a YAML configuration file, rather than by creating a dedicated 
 bridge for each pair of communicating components.
 For a system made of *N* components, this means that the number of new software parts to add grows as *N*
 rather than *N²*.
 
-Notice that :code:`integration-service` already provides the **System-Handle** for some of the most common
+Notice that *eProsima Integration-Service* already provides the **System-Handle** for some of the most common
```

Comment:
I've got the feeling that this is a topic that was partially adressed in the very first "Main features" section of the index, and here you're adding some more information, while mentioning facts that were stated before (for example, the available SystemHandle list (ROS1, ROS2, websocket etc). I don't know if it would be better to maybe extend the information in the "Main features" section, and then here include a summary with references to that section...

Reply:
yes. I see your point.

Reply:
@FranFin